src/Adapters/Storage/Postgres/PostgresClient.js

```diff
@@ -20,7 +20,7 @@ export function createClient(uri, databaseOptions) {
 
   if (process.env.PARSE_SERVER_LOG_LEVEL === 'debug') {
     const monitor = require('pg-monitor');
-    if(monitor.isAttached()) {
```

Comment:
Please revert unrelated changes in postgres client

Reply:
Yeah,  i think @Moumouls  tweaked the  linter at some point. I'm not sure why there is so many deltas. Ill go through one mor etime